test/b1-advanced-security.js

```diff
@@ -347,9 +346,106 @@ describe('b1 - advanced security', function (done) {
 
   });
 
+  var testGroupAdmin = {
+    name: 'TEST GROUP ADMIN',
+
+    custom_data: {
+      customString: 'custom1',
+      customNumber: 0
+    },
+
+    permissions: {
+      methods: {
+        //in a /Mesh name/component name/method name - with possible wildcards
+        '/testadvancedSecurity/security/*': {authorized: true}
+      },
+      events: {
+        //in a /Mesh name/component name/event key - with possible wildcards
+        '/testadvancedSecurity/security/*': {authorized: true}
+      }
+    }
+  };
+
+  var testGroupUser = {
+    name: 'TEST GROUP USER',
+
+    custom_data: {
+      customString: 'custom2',
+      customNumber: 0
+    },
+
+    permissions: {
+      methods: {
+        //in a /Mesh name/component name/method name - with possible wildcards
+        '/testadvancedSecurity/security/*': {authorized: false}
+      },
+      events: {
+        //in a /Mesh name/component name/event key - with possible wildcards
+        '/testadvancedSecurity/security/*': {authorized: false}
+      }
+    }
+  };
+
+  it('update user role to TEST GROUP USER to prevent the user from calling methods from /testadvancedSecurity/security',
+  function(done){
+    this.timeout(2000);
+    var testUser = {
+      username : 'user1',
+      password : 'password',
+      custom_data : {
+        role : 'TEST GROUP ADMIN'
+      }
+    };
+
+    adminClient.exchange.security.addGroup(testGroupAdmin, function (e, result) {
```

Comment:
@JohanvdWest @JensEggers 26/08-ak : Fixed in commit https://github.com/happner/happner/pull/188/commits/4dba96dc5d6f03cb135dac468534fc432684155d